enterprise/server/workflow/service/service.go

```diff
@@ -421,11 +421,11 @@ func (ws *workflowService) providerForRepo(u *url.URL) (interfaces.GitProvider,
 	return nil, status.InvalidArgumentErrorf("could not find git provider for %s", u.Hostname())
 }
 
-func (ws *workflowService) GetWorkflows(ctx context.Context, req *wfpb.GetWorkflowsRequest) (*wfpb.GetWorkflowsResponse, error) {
+func (ws *workflowService) GetWorkflows(ctx context.Context, protoContext *ctxpb.RequestContext) (*wfpb.GetWorkflowsResponse, error) {
 	if err := ws.checkPreconditions(ctx); err != nil {
 		return nil, err
 	}
-	groupID, err := perms.AuthenticateSelectedGroupID(ctx, ws.env, req.GetRequestContext())
+	groupID, err := perms.AuthenticateSelectedGroupID(ctx, ws.env, protoContext)
 	if err != nil {
 		return nil, err
 	}
```

Comment:
I should probably remove this `AuthenticateSelectedGroupID` func at some point - the new way to get the authenticated user's selected group ID is

```
u, err := perms.AuthenticatedUser(ctx)
if err != nil {
    return err
}
groupID := u.GetGroupID()
```

that way you can avoid the awkward `protoContext` param